src/create.rs

```diff
@@ -111,6 +123,8 @@ impl Party1 {
             (X_other, R_other, Y_other),
         );
 
+        let sig_TX_c_self = todo!("pSign TX_c.digest() using x_self and Y_other");
+
         let TX_s = SplitTransaction::new(
```

Comment:
We are slightly deviating from the paper with this optimization. If we were to follow fig.6 in the paper we should introduce an extra step here for generating TX_s. 
It's fine imho, just wanted to point it out :)